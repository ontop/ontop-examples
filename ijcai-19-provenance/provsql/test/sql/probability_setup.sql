\set ECHO none
SET search_path TO public, provsql;

/* Add some probabilities to the personnel table */
ALTER TABLE personnel ADD COLUMN probability DOUBLE PRECISION;
UPDATE personnel SET probability=id*1./10;

SELECT create_provenance_mapping('p', 'personnel', 'probability');

SELECT value FROM p ORDER BY value;
