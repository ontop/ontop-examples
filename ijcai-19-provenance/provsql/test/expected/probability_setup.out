\set ECHO none
 create_provenance_mapping 
---------------------------
 
(1 row)

 value 
-------
   0.1
   0.2
   0.3
   0.4
   0.5
   0.6
   0.7
(7 rows)

